tests/cli/test_validate_json_telemetry_and_dry_run.py

```
@@ -0,0 +1,104 @@
+            "--fix",
+            "--fix-rules",
+            "V-T1,V-T3",
+            "--format=json",
+        ],
+        capture_output=True,
+        text=True,
+    )
+    data = json.loads(res.stdout or "{}")
+    assert "fixes" in data and isinstance(data["fixes"], dict)
+
+    res2 = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)